wot-wg-2023-draft.html

```diff
@@ -296,7 +301,7 @@ <h3>
               at <a href="https://www.w3.org/TR/wot-discovery/">https://www.w3.org/TR/wot-discovery/</a>,
               as a Candidate Recommendation on 19 January 2023.</p>
             </dd>
-            <dt id="thing-description-deliverable" class="spec">Web of Things (WoT) Thing Description</dt>
+            <dt id="thing-description-deliverable" class="spec">Thing Description</dt>
```

Comment:
While you're changing the names, is Thing Description not an "Update" too?

Reply:
Fixed.